IssueTracker/IssueTracker.UnitTest/TestInitialization.cs

```diff
@@ -148,10 +182,10 @@ public static void AssemblyInitialize(TestContext tc)
     [AssemblyCleanup()]
     public static void AssemblyCleanup()
     {
+
```

Comment:
Keep the PR clean. Remove this empty line.

Reply:
I removed all the empty lines.